content/en/docs/home/_index.md

```diff
@@ -22,9 +22,9 @@ overview: >
   Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>).
 cards:
 - name: concepts
-  title: "Understand the basics"
+  title: "Understand Kubernetes"
   description: "Learn about Kubernetes and its fundamental concepts."
-  button: "Learn Concepts"
+  button: "View Concepts"
```

Comment:
I'd like to see this change removed from the PR, however. "View" implies that everyone can see, when many people use screenreaders and the like. Moreover, I don't think it's a substantive improvement from "Learn".

Reply:
There are 8 button links on said page:

- Learn Concepts
- View Tutorials
- Setup Kubernetes
- View Training
- View Tasks
- Contribute to the docs
- Download Kubernetes
- View Reference

All the other "view" buttons except the training one link to a major section of the documentation, as does the "learn" one I updated here. So main motivation was consistency. I think even the vision impaired would appreciate consistent link naming so they can infer said links all link to something they same - a major section of documentation in this case. 

P.S. I'm not a huge fan of the button links as you can probably tell. Just thought this was a minor tweak worth making while at it.

Reply:
@sgpinkus Ah, I see, thanks for explaining this. 

I'm going to drop an LGTM on this then, as verbiage is really just a nitpick at this point.